client/components/theme-collection/style.scss

```diff
@@ -35,30 +97,47 @@
 	align-items: center;
 
 	.theme-collection__see-all {
-		font-size: 0.875rem;
+		font-size: 1rem;
 		font-style: normal;
 		font-weight: 400;
 		line-height: 20px;
 		letter-spacing: -0.15px;
 		padding: 0;
-		margin-right: 24px;
+		margin-right: 0;
+		align-items: flex-end;
+		text-decoration: underline;
```

Comment:
Why have link underlines on very small screens and remove them when it gets bigger? 🤔

Reply:
It's from Figma, although I think it's incorrect.

cc @lucasmendes-design.

Reply:
Let's not block this PR because of this. Once we have all the Discovery styles + functionality implemented let's do an internal walkthrough session and see what we find. That way we can speed up the development process.